src/Benchmarks/Benchmarks.csproj

```diff
@@ -32,6 +38,7 @@
     <Reference Include="BenchmarkComponent.winmd" Condition="$(UseWinmd) == true">
       <HintPath>$(MSBuildThisFileDirectory)..\_build\$(Platform)\$(Configuration)\BenchmarkComponent\bin\BenchmarkComponent\BenchmarkComponent.winmd</HintPath>
       <IsWinMDFile>true</IsWinMDFile>
+		<Visible>True</Visible>
```

Comment:
What does this property control?

Reply:
I believe it shows the winmd file in the VS solution explorer in case we want to investigate it